### PR TITLE
Changes to upgrade script for Eucalyptus

### PR DESCRIPTION
**Pull request by nedbat**
I've updated the upgrade script for Eucalyptus.  I've changed the name of the script, since "migrate" tends to mean databases.  It's now called "upgrade.sh".

### CODE REVIEW

**Comment by nedbat** [comment]
@edx/devops Comments?

**Comment by jibsheet** [comment]
@clintonb Do you know if this was a discovery failure for a bit?

```
TASK: [discovery | run collectstatic] ***************************************** 
failed: [127.0.0.1] => {"changed": true, "cmd": ["make", "static"], "delta": "0:00:00.037113", "end": "2016-07-20 21:56:12.798271", "rc": 2, "start": "2016-07-20 21:56:12.761158", "warnings": []}
stderr: make: ./node_modules/.bin/r.js: Command not found
make: *** [static] Error 127
stdout: ./node_modules/.bin/r.js -o build.js
```

It's causing Ned's change to fail and I want to know if he should just rebase.

File: util/vagrant/upgrade.sh [hunk 1]
```diff
@@ -226,19 +242,52 @@ EOF
...
+    --extra-vars="migrate_db=no" \
+    --skip-tags="edxapp-sandbox,gather_static_assets" \
+    vagrant-$CONFIGURATION-delta.yml
+  cd ../../..
+
+  echo "Migrate to fix oauth2_provider"
+  ${OPENEDX_ROOT}/bin/edxapp-migrate-lms --fake oauth2_provider zero
+  ${OPENEDX_ROOT}/bin/edxapp-migrate-lms --fake-initial
+
+  sudo rm -rf ${OPENEDX_ROOT}/app/forum/.rbenv ${OPENEDX_ROOT}/app/forum/.gem
```
Comment (jibsheet) [inline util/vagrant/upgrade.sh:287]: Do you want to comment here on the reason (you have notes on a number of the other shenanigan steps).

**Comment by clintonb** [comment]
@jibsheet pull the latest from edx/course-discovery master.

**Comment by jibsheet** [comment]
@nedbat I restarted your travis build which should pull a newer discovery docker container and fix the failure.  Changes look good to me with one question about documentation for the future.
👍

**Pull request merged**